apps/music/js/ui/views/mode_manager.js

```diff
@@ -76,9 +76,23 @@ var ModeManager = {
     }
   },
 
-  initView: function(view, callback) {
-    if (!view.isLoaded) {
-      LazyLoader.load(view.path, function() {
+  waitForView: function(mode, callback) {
```

Comment:
I think `waitForView` makes more sense here, since you pass it a "mode", but it returns the mode's "view", so what you're doing is waiting for a view.